controllers/update_pod_config.go

```diff
@@ -137,9 +138,10 @@ func (u UpdatePodConfig) Reconcile(r *FoundationDBClusterReconciler, context ctx
 		}
 	}
 
-	// If any error has happened return the first error
+	// If any error has happened requeue.
+	// We don't provide an error here since we log all errors above.
 	if len(errs) > 0 {
-		return &Requeue{Error: errs[0]}
+		return &Requeue{Message: "errors occurred during update pod config reconcile"}
```

Comment:
nit: I'd prefer to be logging all the errors here as it could help with filtering.

Reply:
We already log all errors above when they occur why should we log the errors again?